interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -203,28 +89,15 @@ namespace cling {
           if (ArgIndexs.test(index)) {
             // Get the argument with the nonnull attribute.
             Expr* Arg = CE->getArg(index);
-            CE->setArg(index,
-                      (clang::Expr*)SynthesizeCheck(Arg->getLocStart(), Arg));
+            Expr* newArg = SynthesizeCheck(Arg->getLocStart(), Arg);
+            CE->setArg(index, newArg);
```

Comment:
Once the call to `Arg->getLocStart()` is gone you could even combine these into one or two statements.